src/components/AddItem.jsx

```diff
@@ -22,79 +23,78 @@ const radioButtonOptions = [
   },
 ];
 
```

Comment:
I think we can go ahead and create a separate utils.js file and import this function from there. Thoughts?

Reply:
Love the utils idea. To avoid it becoming all plus the kitchen sink, see about separating them per need.